sdk/appconfiguration/app-configuration/samples/helloworld.ts

```diff
@@ -30,16 +30,18 @@ export async function run() {
     await client.deleteConfigurationSetting(greetingKey, {});
     console.log(`${greetingKey} has been deleted`);
 
-    await cleanupSampleValues([greetingKey], client);    
+    await cleanupSampleValues([greetingKey], client);
 }
 
 async function cleanupSampleValues(keys: string[], client: AppConfigurationClient) {
     const existingSettings = await client.listConfigurationSettings({
-        key: keys
+        key: keys.join(",")
```

Comment:
Does your sample need to convert the list into a string? It looks like your convenience later does that, and that there's a `keys` field.

Reply:
It doesn't - this one of those little duplicated pieces of code I need to eliminate at some point but for now Ill re-duplicate it.